openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -648,10 +648,59 @@ def build_anim(project_name, asset_name):
     bpy.ops.scene.make_container_publishable(container_name=cam_container.name)
     cam_instance = bpy.context.scene.openpype_instances[-1]
 
+    animated_objects = set()
     for obj in bpy.context.scene.objects:
         if obj.type == "ARMATURE":
-            # Create animation instance
-            variant_name = obj.name[obj.name.find("RIG_") + 4 :].capitalize()
+            # Create animation instance for rig.
+            variant_name = (
+                obj.name[4].upper() + obj.name[5:]
+                if obj.name.find("RIG_") == 0
+                else obj.name[0].upper() + obj.name[1:]
+            )
```

Comment:
Can't you use `lstrip`?
```suggestion
            variant_name = obj.name.lstrip("RIG_").capitalize()
```

Reply:
```python
"RIG_RobertSmith".lstrip("RIG_").capitalize()  # Result: Obertsmith
```

Reply:
Can't you use `container[AVALON_PROPERTY]["asset_name"]` and `container[AVALON_PROPERTY]["name"]` by matching the container the rig belongs to? This looks too hardcoded to me...